src/change_stream.ts

```diff
@@ -659,7 +670,7 @@ function processNewChange(
   return callback(undefined, change);
 }
 
-function processError(changeStream: ChangeStream, error: AnyError, callback?: Callback) {
+function processError(changeStream: ChangeStream<any>, error: AnyError, callback?: Callback) {
```

Comment:
intentionally keeping `<any>` here?

Reply:
Also, should we standardize to just one of T or TSchema in this file as the generic name?

Reply:
I corrected this, and changed the use of T to TSchema, let me know if its 👍